ambry-network/src/main/java/com.github.ambry.network/NetworkClient.java

```diff
@@ -92,6 +92,7 @@ public NetworkClient(Selector selector, NetworkConfig networkConfig, NetworkMetr
    */
   public List<ResponseInfo> sendAndPoll(List<RequestInfo> requestInfos)
       throws IOException {
+    logger.trace("Initiating Send and poll at " + time.milliseconds());
```

Comment:
can we just use `startTime` in this log message? Or do we not want to include the logging time in `networkClientSendAndPollTime`?